common/network-yarn/pom.xml

```diff
@@ -73,6 +73,10 @@
       <artifactId>${hadoop-client-runtime.artifact}</artifactId>
       <version>${hadoop.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
The default is `provided`?

Reply:
I think it's always better to be explicit.

Reply:
+1 with @cloud-fan

Reply:
Thanks. Added provided.